### ISSUE TITLE
deployment-group-status shows DOWN hosts

### ISSUE THREAD

**Issue by gimaker** [spotify/helios#599]
...and it's confusing. It should probably either filter out DOWN hosts, or show the host status as well.

In the example below the non-up-to-date host is actually DOWN but there's no obvious way to tell:

```
$ helios -d <X> deployment-group-status megatest
Name: megatest
Job Id: megatest-service:0.0.1-SNAPSHOT-e7e221e:6d6153c
Status: ACTIVE
Host selectors:
  role = heliosagent

HOST                                    UP-TO-DATE    JOB                                                STATE      
xxxx-heliosagent-a1.xxxx                X             megatest-service:0.0.1-SNAPSHOT-e7e221e:6d6153c    RUNNING    
xxxx-heliosagent-a1-v1438210071.xxxx                  -                                                  -          
xxxx-heliosagent-staffan1.xxxx          X             megatest-service:0.0.1-SNAPSHOT-e7e221e:6d6153c    RUNNING
```

**Comment by davidxia** [comment]
This is a slightly related question. Should `helios status --job jobID` show hosts that are DOWN?

```
JOB ID                                HOST                                       GOAL     STATE            CONTAINER ID    PORTS
helioscanary:0.8.0-74b6afb:3d335f4    host1              START    RUNNING          b4d570e         hm=20000:20000 http=20001:20001
```

`helios hosts host1` will show it as down.

```
HOST                             STATUS        DEPLOYED    RUNNING    CPUS    MEM     LOAD AVG    MEM USAGE    OS                         HELIOS     DOCKER          LABELS
host1    Down 1 day    1           1          1       3 gb    0.96        0.87         Linux 3.13.0-34-generic    0.8.447    1.6.0 (1.18)    helioscanary=true, role=heliosagent
```

**Comment by davidxia** [comment]
fixed by #630 and #624
